go.mod

```diff
@@ -137,10 +137,18 @@ require (
 require github.com/aws/aws-sdk-go-v2/feature/rds/auth v1.3.10
 
 require (
-	cloud.google.com/go v0.110.8 // indirect
-	cloud.google.com/go/compute v1.23.1 // indirect
+	github.com/felixge/httpsnoop v1.0.4 // indirect
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.47.0 // indirect
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.47.0 // indirect
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.19.0 // indirect
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.19.0 // indirect
+)
+
+require (
```

Comment:
Please make sure there are only 2 `require` blocks: one for direct dependencies and one for indirect. It helps keeping the file tidy

Reply:
I'll try the make mod tidy all, but this is not something I manually added

Reply:
Sometimes this happens automatically. Requires manual fixing: merge all blocks into 2, and then do `go mod tidy`

Reply:
okay I manually consolidated the root go mod and the cert tests one, and then did modtidy-all. My latest commit is the result of this.